test/eth-trezor-keyring.test.js

```diff
@@ -1,24 +1,17 @@
-global.window = require('./window.shim');
-global.navigator = require('./navigator.shim');
-global.self = require('./self.shim');
```

Comment:
I'm not sure why these shims were needed in the past, but I'm pretty certain they are not needed now.